packages/@okta/courage-dist/esm/src/courage/views/components/BaseDropDown.js

```diff
@@ -369,7 +369,7 @@ var BaseDropDown = BaseView.extend({
             "column": 327
           }
         }
-      }) : helper)) + "</span><span class=\"icon-dm\"></span></a><div id=\"okta-dropdown-options\" class=\"options clearfix\" style=\"display: none;\"><ul class=\"okta-dropdown-list options-wrap clearfix\"></ul></div>";
+      }) : helper)) + "</span><span class=\"icon-dm\"></span></a><div id=\"okta-dropdown-options\" class=\"options clearfix\"><ul class=\"okta-dropdown-list options-wrap clearfix\"></ul></div>";
```

Comment:
Why is this in the diff?

Reply:
when I upgraded the courage library, it picked up this change as well